elk/extraction/prompt_dataset.py

```diff
@@ -1,4 +1,4 @@
-"""Collator for prompts."""
+"""prompts for prompts."""
```

Comment:
Typo in comment here

Reply:
Good catch, I did a stupid find and replace here